x-pack/plugins/transform/common/api_schemas/common.ts

```diff
@@ -40,7 +40,13 @@ export type TransformIdParamSchema = TypeOf<typeof transformIdParamSchema>;
 
 export interface ResponseStatus {
   success: boolean;
-  error?: any;
+  // FIXME error response should have unified shape
```

Comment:
Is this comment still needed?

Reply:
yep, want to discuss with @walterra in a follow-up :)